packages/styled-components/src/components/primitives/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { usePopper } from 'react-popper';
```

Comment:
this has to be added as a dependency, am I right?

Reply:
As far as I remember we didn't remove these dependencies from package.json of styled-components (I can see this dependency on line 32)